### DISCUSSION TITLE
Github app fail to connect repo to mend developer platform

### DISCUSSION THREAD

**Discussion by HananoshikaYomaru** [renovatebot/renovate#22992] [Request Help]
### How are you running Renovate?

Mend Renovate hosted app on github.com

### If you're self-hosting Renovate, select which platform you are using.

None

### Was this something which used to work for you, and then stopped?

I am trying to get this working for the first time

### Describe the problem

I install github app on this repo which is only a blank Next.js app. It doesn't connect to Mend developer platform at all and neither create any PR.

![CleanShot 2023-06-27 at 09 51 21](https://github.com/renovatebot/renovate/assets/43137033/529ac4ba-eaf0-46b0-975a-dbdb10c77b83)

![CleanShot 2023-06-27 at 09 52 30](https://github.com/renovatebot/renovate/assets/43137033/320f68fc-be06-4272-b4cd-74d49f86ed43)

![CleanShot 2023-06-27 at 09 49 30](https://github.com/renovatebot/renovate/assets/43137033/467171c4-fb85-4612-89a7-6122668cf6cf)

I follow the [guide](https://github.com/renovatebot/tutorial), but if this step fail, there is nothing I can do here.

### Relevant debug logs

<details><summary>Logs</summary>

```
Copy/paste the relevant log(s) here, between the starting and ending backticks
```

</details>

### Have you created a minimal reproduction repository?

Placeholder value, please select the correct response from the dropdown

**Comment by Gabriel-Ladzaretti** [comment]
It seems that you installed the app at `2023-06-27 02:01:28.350000 +00:00`, and the first scan [onboarding](https://github.com/HananoshikaYomaru/renovate-test/pull/1) this repository completed at `2023-06-27 02:02:07.851000 +00:00`.

Is the UI issue still persisting?

**Reply by HananoshikaYomaru** [reply]
oh yes, you are right. Now it works. Thx. But when I encounter this problem, the pending time is definitely more than 1 minute (I think it is around 30 mins). Not sure why is this though.
